packages/vendure-plugin-payment-extensions/src/settle-without-payment-checker.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CustomerService,
+  ID,
+  Injector,
+  LanguageCode,
+  PaymentMethodEligibilityChecker,
+  TtlCache,
+  idsAreEqual,
+} from '@vendure/core';
+
+const fiveMinutes = 5 * 60 * 1000;
+const groupsPerCustomerCache = new TtlCache<ID, ID[]>({ ttl: fiveMinutes });
+let injector: Injector | undefined;
+
+/**
+ * This checker checks if the logged in customer is in a given group.
+ */
+export const settleWithoutPaymentChecker = new PaymentMethodEligibilityChecker({
+  code: 'settle-without-payment-checker',
+  description: [
+    {
+      languageCode: LanguageCode.en,
+      value: 'Checks if the logged-in customer is in given group',
```

Comment:
This is a typo on my side: It doesn't check the logged-in customer anymore, but rather the customer connected to the order